### PR TITLE
infinite diving gear  fixes auto remove diving gear  from body after …

### PR DESCRIPTION
**Pull request by ARSSANTO**
…wait time

it will prevent infinite under water time and it will auto remove diving gear props from body after the Wait time

### CODE REVIEW

**Comment by GhzGarage** [comment]
Not a fan of this i'll find another way

**Comment by evanillaa** [comment]
https://github.com/qbcore-framework/qb-diving/pull/28

**Comment by ARSSANTO** [comment]
i am going to test it now oooooooppppp

File: client/diving.lua [hunk 1]
```diff
@@ -148,28 +149,32 @@ RegisterNetEvent('qb-diving:client:UseGear', function(bool)
...
-            SetPedMaxTimeUnderwater(PlayerPedId(), 2000.00)
+            SetPedDiesInWater(ped, false)
             currentGear.enabled = true
             TriggerServerEvent('qb-diving:server:RemoveGear')
-            ClearPedTasks(PlayerPedId())
+            ClearPedTasks(ped)
             TriggerEvent('chatMessage', "SYSTEM", "error", "/divingsuit to take off your diving suit")
+           
+           
+            Wait(300000) ---autoremove after this time in ms default is 5 mins
```
Comment (yamieboi) [inline client/diving.lua:163]: Changing this to **Wait(60000*5)** would be better.

**Review by yamieboi — APPROVED** [review]

**Pull request closed without merging**